http.go

```diff
@@ -275,13 +275,11 @@ func (hc *HttpCall) ResponseBodyContains(value string) Step {
 // ResponseBodyMatches is a Step that when executed ensures there is
 // a non-nil hc.ResponseBody and errors unless the hc.ResponseBody
 // matches the regular expression parameter.
-func (hc *HttpCall) ResponseBodyMatches(pattern string) Step {
-	return NewNamedStep("ResponseBodyMatches", func() error {
+func (hc *HttpCall) ResponseBodyMatches(pattern *regexp.Regexp) Step {
+	return NewNamedStep(fmt.Sprintf("ResponseBodyMatches(%s)", pattern), func() error {
```

Comment:
I don't think you've tested this ;) `%s` is only for when the arg is a string. You want `%v` here, and below in your `Errorf` call.